src/components/BackButton/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+import ChevronLeft from 'material-ui-icons/ChevronLeft';
+import Typography from 'material-ui/Typography';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import styles from './styles';
+import { AppLocation, RouterPath } from '../../constants';
+
+@withRouter
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state, props) => ({
+  appLocation: state.App.get('appLocation'),
+}), (dispatch, props) => ({
+}))
+export default class BackButton extends React.Component {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    history: PropTypes.object.isRequired,
+    appLocation: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const {
```

Comment:
can we please put these on one line unless they're extremely long? Our files just keep getting longer and longer from small things like this <3 @dwalintukan

Reply:
sure will remember for the future ones. this one already merged in :P